code/lib/cli/src/migrate.ts

```diff
@@ -1,11 +1,31 @@
 import { listCodemods, runCodemod } from '@storybook/codemod';
+import { runFixes } from './automigrate';
+import { bareMdxStoriesGlob } from './automigrate/fixes/bare-mdx-stories-glob';
+import { JsPackageManagerFactory } from './js-package-manager';
+import { getStorybookVersionSpecifier } from './helpers';
 
-export async function migrate(migration: any, { glob, dryRun, list, rename, logger, parser }: any) {
+const logger = console;
+
+export async function migrate(migration: any, { glob, dryRun, list, rename, parser }: any) {
   if (list) {
     listCodemods().forEach((key: any) => logger.log(key));
   } else if (migration) {
+    if (migration === 'mdx-to-csf' && !dryRun) {
+      await runFixes({ fixes: [bareMdxStoriesGlob] });
+      await addStorybookBlocksPackage();
+    }
```

Comment:
Can you explain this a bit?

Reply:
After reading the PR description more carefully I guess this is explained.

I suppose this a trade-off we can make now for 7.0.0, but should cleanup thoroughly when we do the CLI revamp